JavaScript/JavaScriptSDK/Context/PageVisitTimeManager.ts

```diff
@@ -0,0 +1,96 @@
+﻿
```

Comment:
I don't think this should live in the "Context" folder as it is not a context, it's a separate peice of logic specific to PageViews. Context on the other hand means a data attached to any kind of telemetry (page views, metrics, events, exception, ...)